SystemInformer/mainwnd.c

```diff
@@ -100,17 +100,84 @@ BOOLEAN PhMainWndInitialization(
         PhInitializeStringBuilder(&stringBuilder, 100);
         PhAppendStringBuilder2(&stringBuilder, PhApplicationName);
 
-        if (currentUserName = PhGetSidFullName(PhGetOwnTokenAttributes().TokenSid, TRUE, NULL))
+        if (!PhGetIntegerSetting(L"EnableVerboseWindowTitle"))
         {
-            PhAppendStringBuilder2(&stringBuilder, L" [");
-            PhAppendStringBuilder(&stringBuilder, &currentUserName->sr);
-            PhAppendCharStringBuilder(&stringBuilder, L']');
-            if (KphIsConnected()) PhAppendCharStringBuilder(&stringBuilder, L'+');
-            PhDereferenceObject(currentUserName);
+            if (currentUserName = PhGetSidFullName(PhGetOwnTokenAttributes().TokenSid, TRUE, NULL))
+            {
+                PhAppendStringBuilder2(&stringBuilder, L" [");
+                PhAppendStringBuilder(&stringBuilder, &currentUserName->sr);
+                PhAppendCharStringBuilder(&stringBuilder, L']');
+                if (KphIsConnected()) PhAppendCharStringBuilder(&stringBuilder, L'+');
+                PhDereferenceObject(currentUserName);
+            }
+
+            if (PhGetOwnTokenAttributes().ElevationType == TokenElevationTypeFull)
+                PhAppendStringBuilder2(&stringBuilder, L" (Administrator)");
         }
+        else
+        {
+            DWORD myPID = GetCurrentProcessId();
+            WCHAR myPIDStrArr[PH_INT32_STR_LEN_1];
+            PhPrintUInt32(myPIDStrArr, (ULONG)myPID);
+            PhAppendStringBuilder2(&stringBuilder, L" ");
+            PhAppendStringBuilder2(&stringBuilder, myPIDStrArr);
+            PhAppendStringBuilder2(&stringBuilder, L" ");
```

Comment:
This could be cleaner.

```suggestion
            PhAppendFormatStringBuilder(&stringBuilder, L" %lu", GetCurrentProcessId());
```

Also looks like the way you had it you might have an extra space when appending if the driver is connected below?